demos/switch/index.html

```diff
@@ -0,0 +1,68 @@
+    <link rel="stylesheet" type="text/css" href="../../src/components/overviewPanel/css/OverviewPanel.css" />
+    <link rel="stylesheet" type="text/css" href="../../src/components/switch/css/Switch.css" />
+    <link rel="stylesheet" type="text/css" href="css/Switch.css" />
+
+    <script type="text/javascript" src="../../src/lib/jquery/core/js/jquery.js"></script>
+    <script type="text/javascript" src="../../src/lib/jquery/ui/js/jquery-ui.js"></script>
+    <script type="text/javascript" src="../../src/framework/core/js/jquery.keyboard-a11y.js"></script>
+    <script type="text/javascript" src="../../src/framework/core/js/Fluid.js"></script>
+    <script type="text/javascript" src="../../src/framework/core/js/FluidDocument.js"></script>
+    <script type="text/javascript" src="../../src/framework/core/js/FluidIoC.js"></script>
+    <script type="text/javascript" src="../../src/framework/core/js/DataBinding.js"></script>
+    <script type="text/javascript" src="../../src/framework/core/js/FluidView.js"></script>
+    <script type="text/javascript" src="../../src/components/switch/js/Switch.js"></script>
+
+    <!-- required for the Overview Panel -->
+    <script type="text/javascript" src="../../src/framework/core/js/FluidDOMUtilities.js"></script>
+    <script type="text/javascript" src="../../src/framework/core/js/FluidRequests.js"></script>
+    <script type="text/javascript" src="../../src/lib/fastXmlPull/js/fastXmlPull.js"></script>
+    <script type="text/javascript" src="../../src/framework/renderer/js/fluidParser.js"></script>
+    <script type="text/javascript" src="../../src/framework/renderer/js/fluidRenderer.js"></script>
+    <script type="text/javascript" src="../../src/framework/renderer/js/RendererUtilities.js"></script>
+    <script type="text/javascript" src="../../src/components/overviewPanel/js/OverviewPanel.js"></script>
+    <script type="text/javascript" src="../shared/js/overviewWrapper.js"></script>
+
+    <script type="text/javascript" src="js/SwitchDemo.js"></script>
+
+    <title>Switch UI Demo</title>
+</head>
+
+<body>
+    <div class="flc-overviewPanel fl-overviewPanel-container"></div>
+    <section class="row demo-container">
+        <p id="flc-switchUI-label">Light</p>
+        <div class="flc-switchUI fl-switchUI">
+            <span class="flc-switchUI-off fl-switchUI-text" aria-hidden="true"></span>
+            <span class="flc-switchUI-control fl-switchUI-control"><span class="fl-switchUI-controlKnob"></span></span>
+            <span class="flc-switchUI-on fl-switchUI-text" aria-hidden="true"></span>
+        </div>
+
+        <p id="demo-panel" class="demo-panel">
+            <strong class="demo-panel-face" aria-live="polite"></strong>
+            <strong class="demo-panel-text"></strong>
+        </p>
+    </section>
+    <script>
+        fluid.switchUI(".flc-switchUI", {
+            attrs: {
+                "aria-labelledby": "flc-switchUI-label",
+                "aria-controls": "demo-panel"
```

Comment:
Might be helpful to add an explanation for this aria attribute since it's demo specific and not listed on the API doc for `attrs` option.